### PR TITLE
add-on-manager workflow

### PR DESCRIPTION
**Pull request by maleck13**
adds an image build for the add-on manager

### CODE REVIEW

**Review by jasonmadigan — APPROVED** [review]

File: .github/workflows/addon-manger-image.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          username: ${{ secrets.IMG_REGISTRY_USERNAME }}
+          password: ${{ secrets.IMG_REGISTRY_TOKEN }}
+
+      - name: Build and push add-on manager Image
+        id: build-and-push
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          tags: ${{ env.IMG_TAGS }}
+          target: addon_builder
```
[the remark this answers is not in the document]
Comment (maleck13) [reply .github/workflows/addon-manger-image.yaml:57]: I think there are two targets?
`controller_builder` and `addon_builder` these are the targets in the dockerfile

[the remark this answers is not in the document]
Comment (eoinfennessy) [reply .github/workflows/addon-manger-image.yaml:57]: There's a third target called `add-on-manager` on line 40 of the Dockerfile. Is that the target that you want to use for the image that gets pushed to quay.io?

[the remark this answers is not in the document]
Comment (maleck13) [reply .github/workflows/addon-manger-image.yaml:57]: updated on main now

**Pull request merged**